tests/AheadOfTime/Trimming/StaticLinkedFSharpCore_Trimming_Test/StaticLinkedFSharpCore_Trimming_Test.fsproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Since we are then calling check.ps1::CheckTrim -root "StaticLinkedFSharpCore_Trimming_Test" -tfm "net472" , shouldnt net472 be here as well?

Reply:
I believe that is actually replaced by the specified TFM when we do a build and publish.  I expect that net472 only is there because while working on this I built from the command line a lot, and was only interested in one output.  I will however, add net472 for completeness.